deploy/host-agent/agent.py

```
@@ -0,0 +1,672 @@
+                exception_type=type(exc).__name__,
+                completed_at=time.monotonic(),
+            )
+            return
+
+        # Stage: health_wait
+        store.update(operation_id, current_stage="health_wait")
+        port = int(intended_state.get("server_port", BEDROCK_DEFAULT_PORT))
+        health_reached = _wait_for_health("127.0.0.1", port, health_timeout)
+
+        if health_reached:
+            elapsed_msg = f"Server reached healthy state"
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)